third_party/rust/chromium_crates_io/gnrt_config.toml

```diff
@@ -158,6 +140,13 @@ license_files = ['LICENSE']
 [crate.chrono]
 license_files = ['LICENSE.txt']
 
+[crate.cpufeatures]
+extra_kv = { raw_gn = '''
+if (is_apple && target_cpu == "arm64") {
```

Comment:
what's going on with all of these changes to gnrt_config.toml? We normally should not have to make any changes so I'd like to understand why we are doing this

Reply:
cc: @cdesouza-chromium

Reply:
At some point in M127 these conditionals around `cpufeatures` were all changed by gnrt output. We had to go around redoing the rules in `gnrt_config.toml`, as the android/ios builds were broken.